src/Experience.tsx

```diff
@@ -33,6 +34,11 @@ export const Experience: FC<{ debugMode: boolean }> = ({ debugMode }) => {
         }
     }, { collapsed: false });
 
+    useEffect(() => {
+        if (boxes.length > 1) {
+            sounds.current(SoundTypes.BLOCK_STACK);
+        }
+    }, [boxes])
```

Comment:
This could be moved into it's own component. Maybe when we move to zunstand it wont be need to be away from sounds.